charts/connect/values.yaml

```diff
@@ -2,81 +2,243 @@
+  #   - 10.0.0.0/16
+  #   - 1.84.26.4/32
+
+  # loadBalancerIP:
+
+  # The name of Kubernetes Secret containing the 1Password Connect credentials
   credentialsName: op-credentials
+
+  # The key for the 1Password Connect Credentials stored in the credentials secret
   credentialsKey: 1password-credentials.json
+
+  # Contents of the 1password-credentials.json file for Connect.
+  # Can be set be adding --set-file connect.credentials=<path/to/1password-credentials.json>
+  # to your helm install command
   credentials:
+
+  # Base64-encoded contents of the 1password-credentials.json file for Connect.
+  # This can be used instead of connect.credentials in case supplying raw JSON
+  # to connect.credentials leads to issues.
   credentials_base64:
-  tls:
-    enabled: false
-    secret: op-connect-tls
-  dataVolume:
-    name: shared-data
-    type: emptyDir
-    values: {}
+
+  # The 1Password Connect API repository
   imagePullPolicy: IfNotPresent
+
+  # The 1Password Connect version to pull
   version: "{{ .Chart.AppVersion }}"
+
+  # Node selector stanza for the Connect pod
+  # See: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
   nodeSelector: {}
+
+  # 1Password Connect API and Sync Service
   probes:
+    # Denotes whether the 1Password Connect API readiness probe will operate
+    # and ensure the pod is ready before serving traffic
     liveness: true
+    # Denotes whether the 1Password Connect API will be continually checked
+    # by Kubernetes for liveness and restarted if the pod becomes unresponsive
     readiness: true
+
+  # Additional annotations to be added to the Connect API deployment resource.
   annotations: {}
+
+  # Additional labels to be added to the Connect API deployment resource.
   labels: {}
+
+  # Additional annotations to be added to the Connect API pods.
   podAnnotations: {}
+
+  # Additional labels to be added to the Connect API pods.
   podLabels: {}
+
+  # List of tolerations to be added to the Connect API pods.
   tolerations: []
 
+  # 1Password Connect volume shared between 1Password Connect Containers
+  dataVolume:
+    # The name of the shared volume used between 1Password Connect Containers
+    name: shared-data
+    # The type of the shared volume used between
+    # 1Password Connect Containers
+    type: emptyDir
+    # Desribes the fields and values for configuration of
+    # shared volume for 1Password Connect
+    values: {}
+
+  # Determines if HTTPS Port if setup for the 1Password Connect
+  # Services for 1Password Connect API and Sync
+  tls:
+    # Denotes whether the Connect API is secured with TLS
+    enabled: false
+    # The name of the secret containing the TLS key (tls.key) and certificate (tls.crt)
+    secret: op-connect-tls
+
+  # Ingress allows ingress services to be created to allow external access
+  # from Kubernetes to access 1Password Connect pods.
+  # In order to expose the service, use the route section below
+  ingress:
```

Comment:
Please add all new values you've added here to the README Values section.

Reply:
@volodymyrZotov i updated the `README.md` accordingly let me know if any more context needs to be added.